slash-commands/slash-commands.js

```diff
@@ -107,6 +125,22 @@ CandyShop.SlashCommands = (function(self, Candy, $) {
 			}
 		}
 	};
+	
+	
+	/** Function: nick
+	 * Sets Nickname
+	 *
+	 * Parameters:
+	 *    (String) args The name of the room and the optional password, separated by a space
+	 */
+	self.nick = function(args) {
+		if(args !== undefined && args != '') {
```

Comment:
For consistency with the other commands, I’d invert this check and return early with the usage message, this will drop one level of indent for the actual action.